CHANGELOG.md

```diff
@@ -66,6 +67,7 @@
 - Add support for null types when deriving JSON from Sass variables ([#1019](https://github.com/opensearch-project/oui/pull/1019))
 - Fix alignment of left icon for OuiListGroupItem with multiline text ([#1027](https://github.com/opensearch-project/oui/pull/1027))
 - Update new polygon icon svg ([#1068](https://github.com/opensearch-project/oui/pull/1083))
+- Remove max-height for TreeView node ([[#1118](https://github.com/opensearch-project/oui/issues/1118)])
```

Comment:
```suggestion
```

Reply:
This CHANGELOG should be for this [PR](https://github.com/opensearch-project/oui/pull/1134) .

Reply:
good catch thanks